README.md

```diff
@@ -34,11 +34,16 @@ and more.
 
 ### Install with npm
 
-```bash
+```sh
 npm install plotly.js-dist
 ```
 
-and import plotly.js as `import Plotly from 'plotly.js-dist';` or `var Plotly = require('plotly.js-dist');`.
+and import plotly.js as 
+```js
+import Plotly from 'plotly.js-dist'
+// Or in Node,
```

Comment:
```suggestion
// Or using require,
```
It's not just node, there are active browser-based projects still using `require` - like plotly.js itself 😅